apps/web/pages/booking/[uid].tsx

```diff
@@ -480,6 +480,21 @@ export default function Success(props: SuccessProps) {
                         </div>
                       </>
                     )}
+                    {props.paymentStatus && (
+                      <>
+                        <div className="mt-3 font-medium">
+                          {props.paymentStatus.paymentOption === "HOLD"
+                            ? t("complete_your_booking")
+                            : t("payment")}
+                        </div>
+                        <div className="col-span-2 mb-2 mt-3">
+                          {new Intl.NumberFormat(i18n.language, {
+                            style: "currency",
+                            currency: props.paymentStatus.currency,
+                          }).format(props.paymentStatus.amount / 100.0)}
+                        </div>
+                      </>
+                    )}
```

Comment:
Adds the payment information on the booking page
![CleanShot 2023-08-15 at 23 28 55](https://github.com/calcom/cal.com/assets/65426560/e48f1d8a-6679-4e39-966b-b39dedeef0b3)